api/simqueue/tests/test_quotas_router.py

```diff
@@ -1,14 +1,23 @@
 from datetime import date
 from uuid import uuid4
+from uuid import UUID
 from fastapi.testclient import TestClient
 from simqueue.main import app
 from simqueue.oauth import User
 from simqueue.data_models import ProjectStatus
 import simqueue.db
+import json
+import pytz
```

Comment:
`pytz`, `pytest` and `pytest_asyncio` are not used, so the imports can be removed